lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Wei Gao <gaowei3@qq.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_host_facts
+short_description: Gathers facts about remote vmware host
+description:
+    - Gathers facts about remote vmware host
```

Comment:
The description should have a trailing dot. (Only the short_description must not have it)